userspace/libsinsp/parsers.cpp

```diff
@@ -4493,70 +4490,6 @@ void sinsp_parser::parse_setsid_exit(sinsp_evt *evt)
 	}
 }
 
-void sinsp_parser::parse_setpgid_exit(sinsp_evt *evt)
```

Comment:
If we really wanted to, we could preserve the setpgid parsing for the "self" case (first argument with `pid == vpid` or `pid == 0`). Not sure it's worth it.